pkg/tarmak/vault/vault.go

```diff
@@ -214,7 +217,24 @@ func (v *Vault) VerifyInitFromFQDNs(instances []string, vaultCA, vaultKMSKeyID,
 		return err
 	}
 
-	cl := tunnels[0].VaultClient()
+	var cl *vault.Client
+	readyTunnelFunc := func() error {
+		for _, t := range tunnels {
+			if t.Status() != VaultStateErr {
```

Comment:
I know this is just a backport, but I think we might want to check for more cases here. (Sealed, not initialised)

Reply:
This is done afterwards. This means that we can keep using the same tunnel for these checks instead of creating them over again, when the cluster is already in a responsive state (initialized, sealed etc.)